mithril-common/src/entities/protocol_parameters.rs

```diff
@@ -3,6 +3,7 @@ use serde::{Deserialize, Serialize};
 use sha2::{Digest, Sha256};
 
 /// Protocol cryptographic parameters
+// TODO: `Default` should be removed. There is no functional sense in having a default value for this struct.
```

Comment:
Indeed, you can remove it right away :+1:

Reply:
Done ✅